flow/designs/asap7/eth_top/config.mk

```diff
@@ -0,0 +1,50 @@
+$(info [INFO-FLOW] Ethernet Design)
+
+DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+
+export DESIGN_NAME            = eth_top
+export DESIGN_NICKNAME        = eth_top
+export DESIGN                 = eth_top
+
+export VERILOG_FILES          = $(sort $(wildcard $(abspath $(DESIGN_DIR)/../../src/$(DESIGN))/*.v))
+export SDC_FILE               = $(DESIGN_DIR)/constraint.sdc
+export ABC_AREA               = 1
+
+export CORNER                ?= BC
+
+export LIB_FILES             += $($(CORNER)_LIB_FILES)
+export LIB_DIRS              += $($(CORNER)_LIB_DIRS)
+export DB_FILES              += $($(CORNER)_DB_FILES)
+export DB_DIRS               += $($(CORNER)_DB_DIRS)
+export WRAP_LIBS             += $(WRAP_$(CORNER)_LIBS)
+export WRAP_LEFS             += $(WRAP_$(CORNER)_LEFS)
+export TEMPERATURE            = $($(CORNER)_TEMPERATURE)
+export DFF_LIB_FILE = $($(CORNER)_DFF_LIB_FILE)
+
+export ABC_CLOCK_PERIOD_IN_PS = 400
+
+export DESIGN_POWER           = VDD
+export DESIGN_GROUND          = VSS
+
+export DESIGN_DIR
+
+
+export PLATFORM               = asap7
+export DESIGN_PDK_HOME
+
+export CORNER                ?= BC
+
+export PDN_CFG                = $(FOUNDRY_DIR)/openRoad/pdn/grid_strategy-M2-M5-M7.cfg
+
+export CORE_UTILIZATION       = 0.05
+export CORE_ASPECT_RATIO      = 1
+export CORE_MARGIN            = 2
+export PLACE_DENSITY          = 0.60
+
+export DONT_USE_SC_LIB         = $(OBJECTS_DIR)/lib/merged.lib
+
+
+export DIE_AREA    = 0 0 800 800
+export CORE_AREA   = 4 4 796 796
```

Comment:
It makes no sense to set both CORE_UTILIZATION / CORE_ASPECT_RATIO  and DIE_AREA / CORE_AREA (the later will just be ignored).

Reply:
@maliberty If we leave empty default tool will calculate DIE_SIZE and 50% core utilization. Here manual value will over-ride all settings. Consider like its my required DIE_SIZE

Reply:
The options are mutually exclusive. `DIE_AREA` / `CORE_AREA` are completely ignored in the flow if `CORE_UTILIZATION` exists:
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/abca824a44ebfa6dd1dc079831c738f7ec215477/flow/scripts/floorplan.tcl#L45-L59

Reply:
@rovinski Just explored the floorplan.tcl.
1st priority goes to FOOTPRINT, then CORE_UTILIZATION else DIE_AREA. Its may be wrong. We should give priority to user DIE_AREA as well. We can't force user to use default values calculated by tool. Maybe # Initialize floorplan using DIE_AREA/CORE_AREA also fall in elseif condition only. 
Experts take decision on this.

Reply:
I think OpenLane flow uses "relative" variable to decide tool automatically for DIE_AREA. "absolute" will be user input DIE_AREA

Reply:
It doesn't really matter because they are mutually exclusive. Only 1 of them is supposed to be set. We should probably be returning a flow error. But that also means that having more than 1 defined in the design config is unnecessary. Please only use the one that you intend.